### DISCUSSION TITLE
CH01_12. (개발) 도메인 서비스 구현 - Flask, Migration flask_script 가 실행되지 않습니다.

### DISCUSSION THREAD

**Discussion by howdi2001** [tedilabs/fastcampus-devops#32] [Q&A] [answered]
안녕하세요.
해당 강의 실습 도중에, 
$ docker-compose exec backend sh 이후에
# python manager.py db init 

Traceback (most recent call last):
  File "/app/manager.py", line 3, in <module>
    from flask_script import Manager, MigrateCommand
  File "/usr/local/lib/python3.9/site-packages/flask_script/__init__.py", line 15, in <module>
    from flask._compat import text_type
ModuleNotFoundError: No module named 'flask._compat'

이 에러가 발생합니다. 

Flask == 2.0.2
Flask-SQLAlchemy == 2.5.1
SQLAlchemy == 1.4.26
Flask-Migrate == 3.1.0
Flask-Script == 2.0.6
Flask-Cors == 3.0.9
requests == 2.26.0
mysqlclient == 2.0.3
pika == 1.2.0

버전으로 설치 해서 진행중입니다.
답변 부탁드립니다. 감사합니다.

**Comment by posquit0** [comment, marked as answer]
안녕하세요! 죄송하지만, 해당 강의는 제 담당 강의가 아닙니다. ㅠ.ㅠ
패스트캠퍼스에 문의하시면 담당 강사분과 커뮤니케이션하실 수 있습니다! :)
저는 파트 1,4,5,9를 맡고 있습니다.

감사합니다.
